docs/references/api/preferences.rst

```diff
@@ -264,3 +264,31 @@ To illustrate the use of this preference, consider the following scenario where
       "details": "The query affects 14 rows",
       "hint": null
   }
+
+With ``RPC``, the preference is honored completely on the basis of the number of rows returned in the result set of the ``RPC``. This can be useful for complex mutation queries. For instance, when deleting rows in an ``RPC``:
```

Comment:
```suggestion
With ``RPC``, the preference is honored completely on the basis of the number of rows returned in the result set of the ``RPC``. This can be useful for complex mutation queries using [data-modifying statements](https://www.postgresql.org/docs/current/queries-with.html#QUERIES-WITH-MODIFYING). A simple example:
```